### PR TITLE
Fix Bug 1206178, add android/all link to android send to device widget

### PR DESCRIPTION
**Pull request by schalkneethling**
@jpetto r?

@flodolo We have this string already but, not on this page perhaps?

### CODE REVIEW

**Comment by flodolo** [comment]
It's in a shared file (`download_button.lang`), so it will be picked up automatically.

**Comment by schalkneethling** [comment]
Thx @flodolo

File: media/css/firefox/android.less [hunk 1]
```diff
@@ -645,13 +645,24 @@ html[dir="rtl"] #phone-wrapper figure {
...
+            ul.android {
+                text-align: center;
+
                 li {
-                    float: none;
-                    width: 100%;
+
+                    &.android-all,
+                    &.google-play {
+                        float: none;
```
Comment (jpetto) [inline media/css/firefox/android.less]: Small nit (and existed before this PR) - looks like `float: none;` is already specified in [send-to-device.less](https://github.com/mozilla/bedrock/blob/master/media/css/base/send-to-device.less#L84).

**Comment by jpetto** [comment]
Super small nit, but otherwise looks good. r+wc :white_flower:

**Comment by schalkneethling** [comment]
Thanks @jpetto also, have signoff here https://bugzilla.mozilla.org/show_bug.cgi?id=1206178#c18

**Comment by schalkneethling** [comment]
@jpetto updated, thx!

**Pull request merged**